client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -841,10 +863,20 @@ const pulse = keyframes`
 	100% { opacity: 1; }
 `;
 
-const CheckoutSummaryCard = styled.div`
+const CheckoutSummaryCard = styled.div< { isStreamlinedPrice: boolean } >`
 	border-bottom: none 0;
+	${ ( props ) =>
+		props.isStreamlinedPrice &&
+		css`
+			border: none;
+			border-radius: 4px;
+			background: #fff;
+			padding: 28px;
```

Comment:
Nit: Looks like the summary box has a bit of a chin. Maybe a 14px bottom padding would look more equal to the other sides.

Reply:
Thanks, I've removed bottom padding and adjusted form title spacing to match the design.